lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -438,79 +438,104 @@ module Resources
-                  "category": [
-                      {
-                          "coding": [
-                              {
-                                  "system": "http://terminology.hl7.org/CodeSystem/medicationrequest-category",
-                                  "code": "community",
-                                  "display": "Community",
-                                  "userSelected": false
-                              }
-                          ]
-                      },
-                      {
-                          "coding": [
-                              {
-                                  "system": "http://terminology.hl7.org/CodeSystem/medication-statement-category",
-                                  "code": "patientspecified",
-                                  "display": "Patient Specified",
-                                  "userSelected": false
-                              }
-                          ]
-                      }
-                  ],
-                  "reportedBoolean": true,
-                  "medicationCodeableConcept": {
-                      "text": "Misc Prescription (Freetext Medication)"
-                  },
-                  "subject": {
-                      "reference": "Patient/1316024",
-                      "display": "PETERS, TIMOTHY"
-                  },
-                  "encounter": {
-                      "reference": "Encounter/6188499"
-                  },
-                  "authoredOn": "2016-08-02T11:16:30.000-05:00",
-                  "requester": {
-                      "reference": "Practitioner/2050040",
-                      "display": "Macklin, Burt"
-                  },
-                  "dosageInstruction": [
-                      {
-                          "timing": {
-                              "repeat": {
-                                  "boundsPeriod": {
-                                      "start": "2016-08-02T11:16:00.000-05:00"
-                                  }
-                              }
-                          }
-                      }
-                  ],
-                  "dispenseRequest": {
-                      "validityPeriod": {
-                          "start": "2016-08-02T11:16:30.000-05:00"
-                      }
-                  }
-              }
           }
       ]
     }
-  end
+
+    R4_MEDICATION_REQUEST_CREATE ||= {
+      "resourceType": "MedicationRequest",
+      "status": "active",
+      "intent": "order",
+      "doNotPerform": false,
+      "reportedBoolean": true,
+      "medicationCodeableConcept": {
+        "coding": [
+          {
+            "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+            "code": "352362"
+          }
+        ],
+        "text": "Acetaminophen"
+      },
+      "subject": {
+        "reference": "Patient/1316024"
+      },
+      "encounter": {
+        "reference": "Encounter/1621910"
+      },
+      "note": [
+    		{
```

Comment:
convert all tab characters in this file to spaces

Reply:
Fixed.